resin/settings.py

```diff
@@ -43,7 +43,9 @@ class Settings(object):
         # cache time : 1 week in milliseconds
         'image_cache_time': str(1 * 1000 * 60 * 60 * 24 * 7),
         # token refresh interval: 1 hour in milliseconds
-        'token_refresh_interval': str(1 * 1000 * 60 * 60)
+        'token_refresh_interval': str(1 * 1000 * 60 * 60),
+        # requests timeout: 30 seconds in milliseconds
+        'timeout': str(1000 * 30),
```

Comment:
super-small nit: could we switch this round to `str(30 * 1000)` so that it matches the other settings?

Reply:
done